packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -177,7 +184,7 @@ The following customizations aren't supported:
 * **Okta-hosted widget:** The **Help title** link in the **Customized Help Links** section of the Customization page has been removed and isn't supported.
 * **Self-hosted widget:** The **Need help signing in** string has been removed and isn't supported.
 
-See [Okta Sign-in Widget Help](https://github.com/okta/okta-signin-widget/#help-links).
+See [Okta Sign-in Widget: Help links](https://github.com/okta/okta-signin-widget/#help-links).
 
 ### `processCreds` hook
 
```

Comment:
Sorry, I know I already looked at the Registration Inline Hooks section, but I'd remove "by" in the last sentence of line 197